src/components/Nav.vue

```diff
@@ -49,6 +61,9 @@
 
             <hr class="navbar-divider">
 
+            <router-link v-if="isAdmin" to="/admin" class="navbar-item">
```

Comment:
`isAdmin` is not defined

Reply:
Ah yes. That's good then we don't need this menu entry since you asked me to move the "account validation page" entry into its own submenu.
Fixed